vscode-trace-extension/src/trace-explorer/properties/trace-explorer-properties-view-webview-provider.ts

```diff
@@ -12,6 +12,10 @@ export class TraceExplorerItemPropertiesProvider implements vscode.WebviewViewPr
     private _view?: vscode.WebviewView;
     constructor(private readonly _extensionUri: vscode.Uri) {}
 
+    public rerender() {
```

Comment:
I think you can call the lines below in method `updateTraceServerUrl()`. With this you can avoid adding this public API.